templates/commands.conf.j2

```diff
@@ -476,7 +476,7 @@ object CheckCommand "cst_pgsql" {
   vars.my_login = "$credentials$"
 }
 
-object CheckCommand "systemd" {
+object CheckCommand "systemd2" {
```

Comment:
```suggestion
object CheckCommand "custom-systemd" {
```

Lets do it like this - thats a little bit more clear. It similar to what we did previously with the SAS and SMART checks when those were introduced upstream